packages/components/src/VirtualizedList/ListGrid/ListGrid.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
+import { List as VirtualizedList } from 'react-virtualized';
+import RowSelectionRenderer from '../RowSelection';
+import { getRowData } from '../utils/gridrow';
+
+import theme from './ListGrid.scss';
+
+export function NoRow() {
+	return (
+		<div className={'tc-list-no-rows'}>
+			No rows
+		</div>
+	);
+}
+
+function getRowDataFromParent({ parent, index }) {
+	return getRowData(parent, index);
+}
+
+/**
+ * List renderer that accepts a custom row renderer.
+ * The row renderer will create a row element for each collection item.
+ */
+function ListGrid(props) {
+	const {
+		children,
+		collection,
+		id,
+		height,
+		isSelected,
+		rowHeight,
+		rowRenderer,
+		selectionToggle,
+		width,
+	} = props;
+
+	const enhancedRowRenderer = selectionToggle ?
+		RowSelectionRenderer( // eslint-disable-line new-cap
+			rowRenderer,
+			{
+				isSelected,
+				getRowData: getRowDataFromParent,
+			}) :
+		rowRenderer;
```

Comment:
Please remove the eslint-disable comment from that block. I'd also like the ternary to be clearer.